src/binary.js

```diff
@@ -0,0 +1,35 @@
+const { readFile } = require('fs')
+
+const { detect, Runtime } = require('elf-cam')
```

Comment:
Is there an option to target Node `8.3.0` with `wasm-pack`?

`elf-cam` transpiled code uses `TextDecoder`. This was added in `8.3.0`, but as an export of `require('util')`. In Node `11.0.0`, it was added as a global variable. Part of the transpiled code uses it as a global variable, which throws a deprecation warning on Node `<11.0.0`:

```
ExperimentalWarning: The WHATWG Encoding Standard implementation is an experimental API. It should not yet be used in production applications.
```

In production, `zip-it-and-ship-it` is run with Node `12`. However, in the CLI (`netlify build` command), we are using the user's Node.js version, which might be Node `<11`. We plan on deprecating Node `8`, but not Node `10`.

Reply:
I don't think that warning is related to global exports. The transpiled code always use the export from the util library. I put the code in a gist so you can see it:

https://gist.github.com/calavera/0c77feb68fd57ce6ebcd20c4b3592366

Node 8.3.0 always prints that warning using the util library:

![image](https://user-images.githubusercontent.com/1050/100283091-aeeb1400-2f21-11eb-897c-71180bd48518.png)

Reply:
Oh you're correct. :+1: 
This warning is not printed in Node `10.0.0`. Considering [we're now dropping support for Node 8](https://github.com/netlify/cli/issues/1568) in the CLI, we're probably good there.